mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -196,11 +196,15 @@ class Run(Base, HasStruct):
         id = Column(Integer, primary_key=True)
         uid = Column(String(255, collation=SQLCollationUtil.collation()))
         project = Column(String(255, collation=SQLCollationUtil.collation()))
+        name = Column(
+            String(255, collation=SQLCollationUtil.collation()), default="no-name"
```

Comment:
Is this the convention that was used before? That if no name was provided we use `no-name`?

Reply:
Initially I wanted to make this a `nullable=False` column, it requires to define some default, so I added it, but SQLite has some bug that it can't create `nullable=False` column on a table with records (even if you define default), so for now I left the default, after we'll create the column (schema migration) and fill it up (data migration) we'll be able to make it `nullable=False` (another schema migration)